lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/driver/YarnDriverRuntimeRestartManager.java

```diff
@@ -64,11 +66,13 @@ private YarnDriverRuntimeRestartManager(@Parameter(YarnEvaluatorPreserver.class)
                                           final EvaluatorPreserver evaluatorPreserver,
                                           final REEFEventHandlers reefEventHandlers,
                                           final ApplicationMasterRegistration registration,
-                                          final YarnContainerManager yarnContainerManager) {
+                                          final YarnContainerManager yarnContainerManager,
+                                          final RackNameFormatter rackNameFormatter) {
     this.registration = registration;
     this.evaluatorPreserver = evaluatorPreserver;
     this.reefEventHandlers = reefEventHandlers;
     this.yarnContainerManager = yarnContainerManager;
+    this.rackNameFormatter = rackNameFormatter;
```

Comment:
Nope, that was in ``YarnContainerManager``. The need to format rackname only came in this PR.